content/release/latest-stable-release-sabayon-1903.md

```diff
@@ -0,0 +1,124 @@
+title = "Sabayon 19.03 - New stable release"
+description = "19.03 Press release"
+date = "2019-03-31"
+categories = ["releases", "sabayon"]
+tags = ["releases", "sabayon"]
++++
+
+The team behind Sabayon is excited to present you the latest stable release:
+Sabayon 19.03.
+
+Sabayon is a modern and easy to use distribution based on Gentoo,
+which follows a reliable rolling release model.
+
+Please read on or [download](/download/) your flavour :-)
+
+19.03 is a long awaited release, coming with a lot of new features and enhancements:
+
+- New build infrastructure
+- Switch to Dracut for initramfs generation
+- Full disk encryption support
+- Installer switch from Anaconda to Calamares
+- Support for 32 bit UEFI! (Intel Sticks, and so on...)
+- Latest kernel is 4.20
+- Python 3 is default
+- New Entropy features and general bugfixes, including:
+- Better tracking of "automatic" dependencies...
+- ...and new command, equo mark
+- AMDGPU Enhancements such as extended Vega support including Radeon VII
+- AMD Freesync ready - MESA19, xf86-video-amdgpu-19, solid 5.X kernel available soon
+- Nvidia Freesync ready - "Gsync Supported Monitors" can be enabled in nvidia-drivers-418.43 via the nvidia-settings tool
+- Legacy driver deprecation - nvidia-304.XX drivers are now deprecated. Cards older than Geforce 8000 non-GTX series use nouveau.
+- Work in progress for the new wiki
+- We are on Patreon now! [Support the project](https://www.patreon.com/sabayon)
+
+
+## New Build Infrastructure
+
+Most of the changes that we did sadly aren't noticeable from a user perspective, so we want to share
+our happiness about the codebase state.
+
+We've invested time in making our internal tooling better, dramatically important for future enhancements to our rolling release model.
+
+We are using [Mottainai](https://mottainaici.github.io/docs/) in our build infrastructure to automatize build and release
+of Sabayon, allowing us to scale horizontally the load required to build ISOs, packages and more.
+
```

Comment:
@mudler @geaaru what needs to be added? Is there a link? Otherwise I wouldn't think it's as needed; documentation is (or should be) an obvious thing anyway. ;)